charts/logzio-apm-collector/values.yaml

```diff
@@ -0,0 +1,662 @@
+          endpoint: "0.0.0.0:14268"
+    otlp:
+      protocols:
+        grpc:
+          endpoint: "0.0.0.0:4317"
+        http:
+          endpoint: "0.0.0.0:4318"
+    zipkin:
+      endpoint: "0.0.0.0:9411"
+  processors:
+    resourcedetection/all:
+      detectors: [ec2, azure, gcp]
+    tail_sampling:
+      policies:
+        [
+            {
+              name: error-in-policy,
+              type: status_code,
+              status_code: {status_codes: [ERROR]}
+            },
+            {
+              name: slow-traces-policy,
+              type: latency,
+              latency: {threshold_ms: "${SAMPLING_LATENCY}" }
+            },
+            {
+              name: probability-policy,
+              type: probabilistic,
+              probabilistic: {sampling_percentage: "${SAMPLING_PROBABILITY}" }
+            }
+        ]
+    k8sattributes:
+      extract:
+        metadata:
+        - k8s.pod.name
+        - k8s.deployment.name
+        - k8s.namespace.name
+        - k8s.node.name
+        - k8s.statefulset.name
+        - k8s.replicaset.name
+        - k8s.daemonset.name
+        - k8s.cronjob.name
+        - k8s.job.name
+    resource/k8s:
+      attributes:
+      # Rename fields
+      - key: pod
+        action: insert
+        from_attribute: k8s.pod.name
+      - key: kubernetes_node
+        action: insert
+        from_attribute: k8s.node.name
+      - key: kubernetes_namespace
+        action: insert
+        from_attribute: k8s.namespace.name
+      - key: kubernetes_deployment
+        action: insert
+        from_attribute: k8s.deployment.name
+      - key: kubernetes_pod_ip
+        action: insert
+        from_attribute: k8s.pod.ip
+      - key: kubernetes_statefulset
+        action: insert
+        from_attribute: k8s.statefulset.name
+      - key: kubernetes_replicaset
+        action: insert
+        from_attribute: k8s.replicaset.name
+      - key: kubernetes_cronjob
+        action: insert
+        from_attribute: k8s.cronjob.name
+      - key: kubernetes_daemonset
+        action: insert
+        from_attribute: k8s.daemonset.name
+      - key: kubernetes_job
+        action: insert
+        from_attribute: k8s.job.name
+      # Delete old
+      - key: k8s.deployment.name
+        action: delete
+      - key: k8s.pod.name
+        action: delete
+      - key: k8s.namespace.name
+        action: delete
+      - key: k8s.node.name
+        action: delete
+      - key: k8s.pod.ip
+        action: delete
+      - key: k8s.statefulset.name
+        action: delete
+      - key: k8s.replicaset.name
+        action: delete
+      - key: k8s.daemonset.name
+        action: delete
+      - key: k8s.job.name
+        action: delete
+      - key: k8s.cronjob.name
+        action: delete
+    attributes/env_id:
+      # Add env_id to all spans
+      actions:
+        - key: env_id
+          value: ${ENV_ID}
+          action: insert
+    batch: {}
+  service:
+    extensions: [health_check, pprof, zpages]
```

Comment:
Can we try to use https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/extension/storage/filestorage for data persistence? and see how it effects the performance? same goes for the spm collector? maybe a value that can control the storage type